packages/forms/frontend/sirius-components-forms/src/representations/FormRepresentation.tsx

```diff
@@ -129,15 +134,30 @@ export const FormRepresentation = ({
 
   let content = null;
   if (formRepresentation === 'ready') {
-    content = (
-      <Form
-        editingContextId={editingContextId}
-        form={form}
-        widgetSubscriptions={widgetSubscriptions}
-        setSelection={setSelection}
-        readOnly={readOnly}
-      />
-    );
+    if (form.pages.length > 1) {
+      content = (
+        <Form
+          editingContextId={editingContextId}
+          form={form}
+          widgetSubscriptions={widgetSubscriptions}
+          setSelection={setSelection}
+          readOnly={readOnly}
+        />
+      );
+    } else {
+      content = (
+        <div data-testid="page" className={classes.page}>
+          <Page
+            editingContextId={editingContextId}
+            formId={id}
```

Comment:
This is not the identifier of the form but the identifier of the subscription. I'll fix that.